src/main/java/com/criticalsoftware/Announcement.java

```diff
@@ -0,0 +1,30 @@
+package com.criticalsoftware;
+
+import io.quarkus.mongodb.panache.PanacheMongoEntity;
+import lombok.Getter;
+import lombok.Setter;
+import org.bson.types.ObjectId;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+public class Announcement extends PanacheMongoEntity {
+
+    private String id;
```

Comment:
As the id field is already inherited from the parent class (PanacheMongoEntity), it doesn't need to be explicitly declared in this class.